### PR TITLE
Commerce 1726

### PR DESCRIPTION
**Pull request by luca-pellizzon**
@marco-leo

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
should be fixed in upstream

**Comment by marco-leo** [comment]
@ethanbustad merging to fix a bug, feel free to ask changes if needed

File: commerce-service/src/main/java/com/liferay/commerce/internal/order/comparator/CommerceOrderModifiedDateComparator.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import com.liferay.portal.kernel.util.DateUtil;
+import com.liferay.portal.kernel.util.OrderByComparator;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceOrderModifiedDateComparator
+	extends OrderByComparator<CommerceOrder> {
+
+	public static final String ORDER_BY_ASC = "createDate ASC";
```
Comment (ethanbustad) [inline commerce-service/src/main/java/com/liferay/commerce/internal/order/comparator/CommerceOrderModifiedDateComparator.java:27]: @luca-pellizzon shouldn't the value of `ORDER_BY_ASC` be `"modifiedDate ASC"`? Same question for `ORDER_BY_DESC`.

Comment (luca-pellizzon) [reply commerce-service/src/main/java/com/liferay/commerce/internal/order/comparator/CommerceOrderModifiedDateComparator.java:27]: you're right, will fix it asap

**Pull request merged**